api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -118,6 +118,7 @@ message RedisProxy {
   }
 
   message PrefixRoutes {
+    // [#next-free-field: 6]
```

Comment:
v2 of the API is frozen (immutable). Please revert the changes to this file and only apply the changes to the v3 proto.

Reply:
fixed